srml/system/src/lib.rs

```diff
@@ -52,6 +52,28 @@ use runtime_io::{twox_128, TestExternalities, Blake2Hasher};
 #[cfg(any(feature = "std", test))]
 use substrate_primitives::ChangesTrieConfiguration;
 
+/// Handler for when a new account has been created.
+pub trait OnNewAccount<AccountId> {
```

Comment:
Does this and `IsDeadAccount` belong in `system` long-term? Is there a plan to move these to `indices` or some other `accounts-chain` thing?

Reply:
Yes, this is what I mean -- I'm sure for @0x7CFE's UTXO prototype it doesn't help much to have AccountID in System for example, while other things in System are useful